packages/story-editor/src/components/form/color/constants.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const SPACING = {
+  FLOATING_MENU: { x: 0, y: 12 },
+  DEFAULT_SIDEBAR: { x: 230, y: 0 },
+  SIDEBAR_WITHOUT_EYEDROPPER: { x: 208, y: 0 },
+  IS_RTL: { x: 12, y: 0 },
+};
```

Comment:
The `dock` feature in Popups is based on the left of the element we dock to. In the future it would be handy to be able to dock to the right and still have all the RTL, anchor.width, and placement details be able to do the math to render it were we would expect. That is a bit much for this pr. So we can create a follow up ticket to address.

Reply:
Added ticket: #10928

Reply:
Awesome. Thanks for documenting & creating the ticket!

Reply:
Please add any details I may have missed, or that you think would be helpful for whoever tackles this. I might be a little too _in it_ to objectively note all the nuances.